setup.py

```diff
@@ -3,7 +3,6 @@
 from setuptools import setup, find_packages
 
 requirements = [
-    'pulpcore-plugin~=0.1rc1',
```

Comment:
pulp_file for example lists pulpcore~=3.0rc7

Do you think it is best to leave this out?